op-node/rollup/driver/driver.go

```diff
@@ -157,10 +157,12 @@ type Drain interface {
 
 // NewDriver composes an events handler that tracks L1 state, triggers L2 Derivation, and optionally sequences new L2 blocks.
 func NewDriver(
+	driverCfg *Config,
+	interopCfg interop.Setup,
+	rollupCfg *rollup.Config,
```

Comment:
I renamed cfg to rollupCfg because there are now 4 configs to deal with. Can't use the top-level Config without an import cycle against node